tests/configs/gpii.tests.acceptance.localInstall.config.json

```diff
@@ -9,5 +9,5 @@
             }
         }
     },
-    "mergeConfigs": "%universal/gpii/configs/gpii.config.development.all.local.json"
+    "mergeConfigs": "%universal/gpii/configs/gpii.config.all.development.dr.production.json"
```

Comment:
@kaspermarkus I figured it out.  You meant that "gpii.tests.acceptance.**localInstall**.dynamicDeviceReporter.config.json" is identical to "gpii.tests.acceptance.localInstall.confg.json".  I'll delete the latter and replace it with the former, and update docs etc.